concentration/IAButton.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  IAButton.swift
+//  concentration
+//
+//  Created by 1 on 30.01.2023.
+//
+
+import UIKit
+
+class IAButton: UIButton {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        self.layer.cornerRadius = 8
+        self.backgroundColor = .link.withAlphaComponent(0.2)
```

Comment:
maybe put this color in some UI constatnts structure?

Reply:
yep, will do!

Reply:
yep, will do it!